hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/TargetManagementTest.java

```diff
@@ -667,11 +763,14 @@ public void bulkTargetCreationAndDelete() throws Exception {
 
         targetManagement.delete(targetsIdsToDelete);
 
-        final List<Target> targetsLeft = targetManagement.findAll(PageRequest.of(0, 200)).getContent();
+        final List<Target> targetsLeft = targetQueryExecutionManagement.findAll(PageRequest.of(0, 200))
+                .getContent();
         assertThat(firstList.spliterator().getExactSizeIfKnown() - numberToDelete).as("Size of split list")
                 .isEqualTo(targetsLeft.spliterator().getExactSizeIfKnown());
 
-        assertThat(targetsLeft).as("Not all undeleted found").doesNotContain(deletedTargets);
+        assertThat(targetsLeft).as("Not all undeleted found").doesNotContainAnyElementsOf(targetsToDelete);
```

Comment:
Local variable `deletedTargets` in line 763 is not needed anymore thus can be removed